src/Satooshi/Bundle/CoverallsV1Bundle/Api/Jobs.php

```diff
@@ -152,7 +152,7 @@ protected function upload($url, $path, $filename)
             'multipart' => [
                 [
                     'name' => $filename,
-                    'contents' => fopen($path, 'r'),
+                    'contents' => file_get_contents($path),
```

Comment:
@KazuakiM , root evil of all test crashing was that `fopen`, which got **not** `fclosed` under mocked guzzle client